Makefile.include

```diff
@@ -726,6 +726,7 @@ termdeps: $(TERMDEPS)
 term: $(TERMDEPS)
 	$(call check_cmd,$(TERMPROG),Terminal program)
 	$(TERMPROG) $(TERMFLAGS) $(TERMTEE)
+	$(POSTTERM)
```

Comment:
Question to the testing people @fjmolinas @MrKevinWeiss, @leandrolanzieri ...: is this fine?